src/map/map.c

```diff
@@ -3769,8 +3769,29 @@ static int map_waterheight(char *mapname)
 	// read & convert fn
 	rsw = grfio_read(fn);
 	if (rsw) {
+		if (memcmp(rsw, "GRSW", 4) != 0) {
+			ShowWarning("Failed to find water level for %s (%s)\n", mapname, fn);
+			aFree(rsw);
+			return NO_WATER;
+		}
+		int majorVersion = rsw[4];
+		int minorVersion = rsw[5];
```

Comment:
lower case for all words is very bad idea.
first letter must be small. But if write all words without separator and same case, variable name can be unreadable. This mean need write or ``majorVersion`` or ``major_version`` but first i think is better

Reply:
@4144 I think `major_version` would be appropriate here since all the other variables in the function are lower case and in the other one too ex `water_height`.

Reply:
done